libraries/apollo-mockserver/src/appleMain/kotlin/com/apollographql/apollo3/mockserver/AppleMockServer.kt

```diff
@@ -111,12 +111,16 @@ actual class MockServer(
     queueMockServerHandler.enqueue(mockResponse)
   }
 
+  override suspend fun closeSynchronously() {
+    close()
+  }
+
   /**
-   * [MockServer] can only stop in between complete request/responses pairs
+   * [AppleMockServer] can only stop in between complete request/responses pairs
    * If stop() is called while we're reading a request, this might wait forever
    * Revisit once okio has native Timeout
    */
-  actual override suspend fun stop() {
+  override fun close() {
```

Comment:
`MockServer` now implements `Closeable` because it's quite convenient. The semantics arent really well defined yet (same as before). Some targets shut down synchronously, others asynchronously. Some shut down existing connections, other are more graceful. 

There is still `closeSynchronously` as `@ApolloExperimental` the time we fine tune this.